### ISSUE TITLE
Unable to link to relative file path

### ISSUE THREAD

**Issue by sahil48** [jgraph/drawio#2256]
* [ X ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ X ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
I need to create a link to a relative file path so a link in an exported pdf to another file will work on multiple Windows computers and iOS devices regardless of the filesystem.

For example, instead of C:\Users\User\Dropbox\folderdrawio\Document 1.docx, if the exported pdf or drawio file is in Dropbox, I need the file path in the flowchart to be file://folderdrawio/Document1.docx

**Describe the solution you'd like**
Allow export of file://{relative-file-path} so relative links in pdf documents open properly. Currently, I have not figured out how to do this.

**Comment by partypointer** [comment]
- [X] I agree to follow the Code of Conduct that this project adheres to.
- [X] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

Hi,
I'm an old time user of draw.io and I've tried to link to a certain page in my document when it is exported to pdf format.
While the feature works when linking to a static URL, it doesn't really work to navigate a document itself.

I've tried using the link to a page function, however it doesn't seem to work as intended when exported to pdf format.

The only thing that worked to some extent was to place two dots at the start of the link.
For example:
`..#page=4`
Should have worked to go to page 4 of the document when opened in a web browser.
However, what it did was sending me to a different URL in my File System:
`file:///C:/myPrograms/drawio/draw.io/resources/#page=4`

This file seems to be something from the draw.io's installation folder, which obviously isn't the pdf it should be referencing.
I think this happens because when exporting to pdf, draw.io sets the current path to a certain installation file, however I could be wrong.

I do not currently have a solution to this problem, however I hope to have given some kind of hint towards a possible fix.

Hopefully someone will fix this thing which is pretty annoying to me and possibly many others.
